features/abonnementen-service/registreer-abonnee.feature

```diff
@@ -1,30 +1,56 @@
-#language: nl
-
+# language: nl
 Functionaliteit: Registreer abonnee
-
   Als consumer van BRP API Gebeurtenissen
```

Comment:
Ik zou de functionaliteit nu anders formuleren, namelijk
```
Als consumer van BRP API Gebeurtenissen
wil ik voor mijn interne afnemers (applicaties, processen) elk een  abonnee kunnen registreren
zodat ik niet zelf hoef te regelen dat zij de gebeurtenissen krijgen van personen aan wie zij diensten verlenen 
```

En hetzelfde met de regels (requirements)

```
Regel: Bij het registreren van een abonnee moet een unieke en valide naam worden opgegeven
Regel: Alleen gemeentes kunnen abonnees registreren
```

Naar mijn mening ligt de nadruk veel te veel in het schrijven van scenarios (voor het valideren/testen) dan in het helder krijgen van de requirements en het doel van een functionaliteit